data/data_test.go

```diff
@@ -108,6 +148,101 @@ func TestGetAvailableVersions(t *testing.T) {
 	assert.Equal(t, versions, mockVersions, "component versions data")
 }
 
+func TestGetCluster(t *testing.T) {
+	mockCluster := getMockCluster(t)
+	cluster, err := GetCluster(mocks.InstalledMockData{}, &mocks.ClusterIDMockData{}, mocks.LatestMockData{})
+	assert.NoErr(t, err)
+	assert.Equal(t, cluster.ID, mockCluster.ID, "ID value")
+	for i, component := range cluster.Components {
+		// verify that GetCluster returned the expected mock Component data
+		assert.Equal(t, component.Component, mockCluster.Components[i].Component, "Component value")
+		// verify that GetCluster returned the expected mock Version data
+		assert.Equal(t, component.Version, mockCluster.Components[i].Version, "Version property")
+		// TODO
+		/*latestComponent := getMockLatest(component.Component.Name, t)
+		// verify that the expected "UpdateAvailable" field values were added
+		assertUpdateAvailableAdded(latestComponent, cluster.Components[i], t)*/
+	}
+}
+
+func TestAddUpdateData(t *testing.T) {
+	mockCluster := getMockCluster(t)
+	// AddUpdateData should add an "UpdateAvailable" field to any components whose versions are out-of-date
+	err := AddUpdateData(&mockCluster, mocks.LatestMockData{})
+	assert.NoErr(t, err)
+	//TODO when newestVersion is implemented, actually test for the addition of "UpdateAvailable" fields
```

Comment:
Might suggest adding the github ticket that tracks the implementation (and tests for) newestVersion here...